### PR TITLE
adds the meat clown

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

this pr adds the meat clown! you can craft it with bananas and meat, its slippery
sprites by @Onule 

## Why It's Good For The Game

new epic food, and its really weird that our game with clowns doesnt have an iconic clown food

## Changelog
:cl: Fikou, Onule
add: Meat Clown
/:cl:

### CODE REVIEW

**Comment by trollbreeder** [comment]
clown

**Comment by gitbirb** [comment]
i love this

**Comment by wesoda25** [comment]
meat clown finna make me act up

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm [hunk 1]
```diff
@@ -201,7 +201,7 @@
 	subcategory = CAT_MEAT
 
 /datum/crafting_recipe/food/ricepork
-	name = "Rice and pork"
+	name = "Rice and Pork"
```
Comment (ATH1909) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:204]: undocumented change smh my head my head

**Comment by Moonman121** [comment]
Finally some decent content.

**Comment by RaveRadbury** [comment]
I was expecting
https://www.youtube.com/watch?v=k0oYjzU-t_0

**Comment by Tharcoonvagh** [comment]
Meat clown and tesloose

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:204]: shyt uip

**Comment by Megarop** [comment]
> new epic food, and its really weird that our game with clowns doesnt have an iconic clown food
What about the clown burger, clown cake, clown snowcone, honkdae, or clown's tears (if you count that as food and not a drink)?

**Review by Twaticus — APPROVED** [review]
thanks i hate it

**Comment by SumFaggotPlayTester** [comment]
Mildly disturbing but also delicious.

**Pull request merged**